.cspell/balena-words.txt

```diff
@@ -0,0 +1,27 @@
+balena
+balena-api
+balena-base-ui
+balena-build
+balena-builder
+balenahup
+balenalib
+balenistas
+bindmount
+boostrap
```

Comment:
Is this really a "balena word", or is it a typo for `bootstrap` ?

Reply:
> Making the dictionaries perfect will be a group effort.

I'm not gonna submit PRs, I don't work for Balena. I'm just trying to be helpful by pointing out things that my eye-for-detail spotted :eyes: :slightly_smiling_face: